### PR TITLE
Added hungarian translation

### PR DESCRIPTION
**Pull request by titanicbobo**

### CODE REVIEW

File: app/src/main/res/values-hu/strings.xml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    <string name="sensor_error_message">Az érzékelő nem elérhető</string>
+    <string name="sensor_calibration_text">Az érzékelő pontosságát bármikor megpróbálhatja javítani a készülék 8-as alakban történő forgatásával.</string>
+
+    <string name="location_loading">Várakozás a helyadatok frissítésére</string>
+    <string name="location_error_message">A helyadatok nem elérhetőek</string>
+    <string name="location_not_present">A tartózkodási helyet nem lehetett meghatározni. A mágneses észak használata.</string>
+    <string name="location_reload">Tartózkodási hely frissítése</string>
+    <string name="access_location_permission_rationale_title">Engedélyre van szükség a tartózkodási helyéhez való hozzáféréshez</string>
+    <string name="access_location_permission_rationale_message">A valódi északra való igazításhoz ennek az alkalmazásnak meg kell határoznia a mágneses deklinációt az aktuális helyzete alapján.</string>
+    <string name="access_location_permission_denied">Missing permission to access location. Falling back to magnetic north.</string>
```
Comment (Kr0oked) [inline app/src/main/res/values-hu/strings.xml:59]: access_location_permission_denied not translated

File: app/src/main/res/values-hu/strings.xml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    <string name="source_code">Forráskód</string>
+    <string name="version">Verzió</string>
+
+    <string name="error_message">Hiba történt.\nÜzenet: %1$s\nHibakód: %2$s</string>
+
+    <string name="error">Hiba</string>
+    <string name="ok">OK</string>
+    <string name="no_thanks">Nem, köszönöm</string>
+
+    <string name="source_code_name" translatable="false">GitHub</string>
```
Comment (Kr0oked) [inline app/src/main/res/values-hu/strings.xml:87]: You can delete all the strings with translatable="false" from your file.

**Review by Kr0oked — CHANGES_REQUESTED** [review]
Thanks for providing the hungarian translation.
I really appreciate that.
There are two things in the strings.xml that we could improve.

And could you also provide translations for the texts of the app store? You can find them in the fastlane/android directory. You just have to provide translated versions for the files full_description.txt, short_description.txt and title.txt like in the other directories there . I will take care of the rest.

Thanks in advance!
Regards

**Comment by titanicbobo** [comment]
Thank you for your feedback! I fixed up those things you requested in strings.xml and I added those three files translated files to fastlane.

**Review by Kr0oked — APPROVED** [review]
Thanks for doing the adjustments.
The translation will be included in the next release, but there is not yet a plan when that will be.

**Pull request merged**